package.json

```diff
@@ -1,15 +1,11 @@
 {
 	"scripts": {
 		"build": "npm run -w assets build",
-		"prebuild:dev": "npm run remove-dist",
 		"build:dev": "npm run -w assets build:dev",
-		"prebuild:production": "npm run remove-dist",
 		"build:production": "npm run -w assets build:production",
 		"build:test": "npm run -w assets build:test",
 		"watch": "npm run -w assets watch",
-		"prebuild:storybook": "npm run remove-dist",
 		"build:storybook": "npm run -w storybook build",
-		"remove-dist": "rimraf ./dist",
```

Comment:
We need to remove `rimraf` from this `package.json` and add it in workspaces where we use it now.